audit/projects/k8s-infra-e2e-boskos-050/services/compute/project-info.json

```diff
@@ -68,7 +68,7 @@
       "metric": "URL_MAPS"
     },
     {
-      "limit": 30,
+      "limit": 150,
```

Comment:
@ameukam was this anything you did manually? we had an open issue about differing quotas somewhere, but I felt it wasn't worth reconciling if there was no automatable way of doing so

Reply:
I did nothing. I thought he was you or boskos "magically" increased the limits. I just know we got some alerts recently related about resource exhaustion : https://kubernetes.slack.com/archives/C7J9RP96G/p1628134230006600. Only someone with access to those alerts could identity which projects are concerned.